### PR TITLE
fix h types

### PR DESCRIPTION
**Pull request by dancespiele**
everything is explain here https://github.com/jorgebucaran/ultradom/pull/105

### CODE REVIEW

**Comment by dancespiele** [comment]
I update this according with this https://github.com/jorgebucaran/ultradom/pull/105

File: hyperapp.d.ts [hunk 1]
```diff
@@ -21,6 +21,11 @@ export interface Component<Attributes = {}> {
   (attributes: Attributes, children: Array<VNode | string>): VNode<Attributes>
 }
 
+/**
+ * Possibles children types
+ */
+export type Children = VNode | string | number | null;
```
Comment (frenzzy) [inline hyperapp.d.ts]: ..and function (subview) #639

Comment (jorgebucaran) [reply hyperapp.d.ts]: I think that belongs to the definition of VNode.

Comment (frenzzy) [reply hyperapp.d.ts]: Why then `string`, `number` and `null` are not VNode definitions?

Virtual Tree:
```js
{ // VNode
  key,
  nodeName,
  attributes,
  children: [
    { key, nodeName, attributes, children }, // VNode
    "string", // text node
    123, // text node
    null, // currently not supported
    undefined, // currently not supported
    () => {}, // subview (just function)
    () => ({ key, nodeName, attributes, children }), // subview may return VNode
    () => "string", // subview may return text node
    () => 123, // subview may return text node
    () => null, // currently not supported
    () => undefined, // currently not supported
  ]
}
```

**Comment by jorgebucaran** [comment]
Better than nothing! I'll merge and work on them later myself. Let's see if I can figure it out.

**Pull request merged**